pkg/proxy/ipvs/README.md

```diff
@@ -47,6 +47,18 @@ kubeadm init --feature-gates=SupportIPVSProxyMode=true
 
 to specify the ipvs mode before deploying the cluster if want to run kube-proxy in ipvs mode.
 
+If you are using kubeadm with a configuration file, you can specify the ipvs mode by adding `featureGates` field.
+Then the configuration file will be look like:
```

Comment:
`The configuration file is similar to:` or `The configuration file looks like:` ?